api/newsletter.ts

```diff
@@ -0,0 +1,49 @@
+import { NowRequest, NowResponse } from '@vercel/node'
+import fetch from 'node-fetch'
+
+const API_KEY = process.env.ECOMAIL_API_KEY || ''
+const API_URL = 'https://api2.ecomailapp.cz/lists/2/subscribe'
+
+export default async (request: NowRequest, response: NowResponse) => {
+  const body = request.body
+
+  if (!body || !body.email) {
+    response.status(404).send('Email is missing')
```

Comment:
Definoval bych tyto message jako klíče nahoře, pak se v tom lépe orientujem

např.
```
const MISSING_EMAIL_ERROR = 'Email is missing';
const UNEXPECTED_ERROR = 'Unexpected error';
```
atd ...

Reply:
Tady ještě váhám, jestli nenahradit ten "Unexpected error" a nepoužít spíš "Network error" - bohužel dokumentace fetch() říká "... it will only reject on network failure or if anything prevented the request from completing." Takže se nějak nedokážu rozhodnout co je přiléhavější.